src/hpa.c

```diff
@@ -479,10 +479,7 @@ hpa_try_hugify(tsdn_t *tsdn, hpa_shard_t *shard) {
 
 	/* Make sure that it's been hugifiable for long enough. */
 	nstime_t time_hugify_allowed = hpdata_time_hugify_allowed(to_hugify);
-	nstime_t nstime;
-	shard->central->hooks.curtime(&nstime, /* first_reading */ true);
-	nstime_subtract(&nstime, &time_hugify_allowed);
```

Comment:
As observed by @Lapenkov, this may trigger 
`
<jemalloc>: src/nstime.c:118: Failed assertion: "nstime_compare(time, subtrahend) >= 0"
`
such as in https://app.travis-ci.com/github/jemalloc/jemalloc/jobs/552738061